x/ecocredit/server/core/query_balances.go

```diff
@@ -0,0 +1,49 @@
+package core
+
+import (
+	"context"
+
+	api "github.com/regen-network/regen-ledger/api/regen/ecocredit/v1"
+	"github.com/regen-network/regen-ledger/types/ormutil"
+	"github.com/regen-network/regen-ledger/x/ecocredit/core"
+
+	"github.com/cosmos/cosmos-sdk/orm/model/ormlist"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+func (k Keeper) Balances(ctx context.Context, req *core.QueryBalancesRequest) (*core.QueryBalancesResponse, error) {
+	acc, err := sdk.AccAddressFromBech32(req.Account)
+	if err != nil {
+		return nil, err
+	}
+
+	pg, err := ormutil.GogoPageReqToPulsarPageReq(req.Pagination)
```

Comment:
not directly related to this PR, I think this function name should have "Slow" suffix.

Reply:
yeah that makes sense - it does call another `slow` method under the hood